File: submissions/codersalive/problem5.cpp
/* package codechef; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		String s=br.readLine();
		s=s.toUpperCase();
		for(int i=0;i<s.length();i++)
		{
		    if(s.charAt(i) == 'A' || s.charAt(i) == 'E' || s.charAt(i) == 'I' || s.charAt(i) == 'O' || s.charAt(i) == 'U' ){
		    System.out.println(s.substring(i,s.length())+s.substring(0,i)+"AY");
		    break; }
		    
		}
	}
}

File: submissions/White Walkers/problem5.java
import java.io.*;
class white_walkers
{
public static void main(String args[])throws IOException
{
BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
int l,i;
String s,s1,t,t1;t1="";s="";t="";s1="";
char ch;ch=' ';
System.out.println("Enter a word:");
s1=br.readLine();
s=s1.toUpperCase();
l=s.length();
for(i=0;i<l;i++)
{
ch=s.charAt(i);
if(ch=='A'||ch=='E'||ch=='I'||ch=='O'||ch=='U')
{
t=s.substring(i,l)+t1+"AY";
break;
}
else
{
t1=t1+ch;
ch=' ';
}
}
System.out.println(t);
}
}
